sdk/storage/azure-storage-blob/src/samples/README.md

```diff
@@ -27,7 +33,6 @@ These sample can be run in your IDE with default JDK.
 6. [Azure Identity Examples][samples_identity]: Use `DefaultAzureCredential` to do the authentication.
 
 ## Troubleshooting
-### General
 When interacting with blobs using this Java client library, errors returned by the service correspond to the same HTTP
 status codes returned for [REST API][error_codes] requests. For example, if you try to retrieve a container or blob that
 doesn't exist in your Storage Account, a `404` error is returned, indicating `Not Found`
```

Comment:
Can't comment properly - in the links section there is the same typo in the link name for SDK_README_GETTING_STARTED